google_guest_agent/network/manager/manager.go

```diff
@@ -227,6 +228,9 @@ func SetupInterfaces(ctx context.Context, config *cfg.Sections, mds *metadata.De
 
 	logger.Infof("Finished setting up %s", activeService.manager.Name())
 
+	// Setup might not have finished when we log and collect this information. Adding this
+	// temporary sleep for debugging purposes to make sure we have up-to-date information.
+	time.Sleep(2 * time.Second)
```

Comment:
since we are only interested on logging and the only shared data is the context we could use a separate go routine and not block here, what do you think?

Reply:
Done